app/src/main/java/com/eveningoutpost/dexdrip/Services/SnoozeOnNotificationDismissService.java

```diff
@@ -42,4 +60,10 @@ protected void onHandleIntent(Intent intent) {
 
         AlertPlayer.getPlayer().Snooze(getApplicationContext(), snooze);
     }
+    
+    private void snoozeOtherAlert(String alertType) {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+        int snoozeMinutes = MissedReadingService.readPerfsInt(prefs, "other_alerts_snooze", 20);
+        UserNotification.snoozeAlert(alertType, snoozeMinutes);
```

Comment:
This is correct that there is a functional behavior change here. In the case that you mention, the next alert will be after 55 minutes.
I think that this is ok, since usually, when alert is snoozed the snooze time will be calculated from current time.
Until my change, there was no way to snooze the other alerts, and I think that this is something that is needed.
Still, if we fill that we don't want any behavior change, I can easily change that.